modules/ob-securely-referencing-string-parameters.adoc

```diff
@@ -0,0 +1,75 @@
+// This module is included in the following assembly:
+//
+// builds/configuring-build-strategies.adoc
+
+:_content-type: CONCEPT
+[id="ob-securely-referencing-string-parameters_{context}"]
+= Securely referencing string parameters
+
+You can reference string parameters by using the `$(params.your-parameter-name)` syntax in your build strategy steps. You can use string parameters when defining environment variables, arguments, or images in a `BuildStrategy` or `ClusterBuildStrategy` CR. 
+
+[NOTE] 
+====
+You can also reference system parameters and strategy parameters by using the `$(params.your-parameter-name)` syntax in your build strategy steps. 
+====
+
+
+In the pod, all `$(params.your-parameter-name)` syntaxes will automatically be replaced by actual strings. However, you must pay attention when you reference a string parameter in an argument by using an inline script. For example, to securely pass a parameter value into an argument defined with a script, you can choose one of the following approaches:
```

Comment:
@shipsing There's nothing incorrect about using the plural "syntaxes", but $-anything defines a variable, right? I am putting the question to you since that's my understanding....